CHANGES.md

```diff
@@ -48,6 +48,9 @@
 - odoc rules now about `ODOC_SYNTAX` and will re-run accordingly (#6010, fixes
```

Comment:
```suggestion
- odoc rules now about `ODOC_SYNTAX` and will rerun accordingly (#6010, fixes
```
I know it looks strange, but rerun is written without the hyphen.

Reply:
Let's fix this in another PR since I didn't introduce that here.

Reply:
I've opened #6155.

Reply:
Thanks! Approved that one.